.travis.yml

```diff
@@ -42,11 +42,6 @@ install:
 - pushd $TRAVIS_BUILD_DIR
 - source scripts/setup.sh || exit 1
 - source scripts/setup_gae.sh || exit 1
```

Comment:
Yep, I think it makes sense to do that, but I agree we should leave this for a separate PR.